docs/adr/02_DNS.md

```diff
@@ -0,0 +1,38 @@
+# DNS
+
+## Aims
+
+* delta.communities.gov.uk, reporting.communities.gov.uk and likely a couple of others pointing at our CloudFront distribution(s)
+  * And similar for delta.staging.communities.gov.uk etc.
+* No regular changes required from DLUHC
+  * So automated SSL certificate renewal for said CloudFront distribution
+  * Internal changes to our infrastructure shouldn't require changes from DLUHC
+* DLUHC maintain control of the main communities.gov.uk domain, so no delegating it or wildcard subdomains
+* We can create SSL certificates for our ALBs (without DLUHC input)
+* Be able to test the sites before switching DNS during go live
+
+## Proposed records from DLUHC
+
+* `infra.communities.gov.uk` delegated to Route53 in this account
+* ACM validation record for a wildcard certificate on `*.communities.gov.uk`
```

Comment:
While that would be convenient, they might not go for this. The current cert on delta.communities.gov.uk is issued to delta.communities.gov.uk specifically.

Reply:
Yeah they might not, but it's worth a try. From trying it out ACM requires separate validation records for each SAN in the cert, even if they're all subdomains of a common domain, so if they want to avoid wildcard certs they'll have to set up a validation record for each one.